common/endpoint-shared/src/main/java/org/kaaproject/kaa/common/avro/GenericAvroConverter.java

```diff
@@ -156,7 +157,8 @@ public String encodeToJson(T record) throws IOException{
      */
     public byte[] encodeToJsonBytes(T record) throws IOException{
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
-        jsonEncoder = EncoderFactory.get().jsonEncoder(this.schema, baos);
+        boolean base64 = true;
+        jsonEncoder = EncoderFactory.get().jsonEncoder(base64, this.schema, baos);
```

Comment:
Don't declare base64 variable, just call jsonEncoder method with true as the last argument